README.md

```diff
@@ -47,7 +47,7 @@ make boot
 
 You may need to install the following (unless you already have it):
 * GRUB2 bootloader tools - `grub2-common` package (e.g. `apt install grub2-common`)
-* ISO generation tools - `xorriso` package (e.g. `apt install xorriso`)
+* ISO generation tools - `xorriso` and `mtools` package (e.g. `apt install xorriso mtools`)
 
```

Comment:
`mtools` is needed for creating the ISO via `grub-mkrescue`? If so, you should mention that in the commit log, at least. However, it should already had been installed via the `grub2-common` package, as that depends on `grub-common` which "suggests" `mtools`. But yeah, it's not a strong dependency, so we may want to make it explicit.